src/main.rs

```diff
@@ -1,20 +1,49 @@
+use rand::seq::SliceRandom;
+use rand::thread_rng;
+use std::time::{SystemTime, UNIX_EPOCH};
+
 fn main() {
-    println!("Hello, world!");
-    println!("Hello, world!");
+    let now = SystemTime::now();
+    match now.duration_since(UNIX_EPOCH) {
+        Ok(duration) => {
+            let seconds_since = duration.as_secs();
+            if seconds_since % 5 == 0 {
+                println!("UWUUUUUUUUUUUUUUUUUUUUU");
+            } else {
+                println!("YOU HAVE COMMITTED HERESY!");
+            }
+        }
+        Err(_) => {
```

Comment:
Implement a custom Error enum that impls Error from the std to handle this, and remove the direct error call in favor of a question mark

Reply:
Job done